homeassistant/helpers/config_validation.py

```diff
@@ -402,3 +402,9 @@ def validator(value):
     [vol.Any(SERVICE_SCHEMA, _SCRIPT_DELAY_SCHEMA, EVENT_SCHEMA,
              CONDITION_SCHEMA)],
 )
+
+
+def string_schema(*args: Sequence) -> Dict:
+    """Return a schema dict of Required values or Optional values."""
+    return {val if isinstance(val, vol.Optional) else vol.Required(val): string
+            for val in args}
```

Comment:
Since we're not getting consensus, let me roll it back for this PR

But, we do save 220 lines (we don't save 390 lines) and all those `vol.Required` classes is created at load time in any case, very similar to the function